src/gaussian_plot/ui.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="GaussianPlot" parent="GtkWidget">
+    <property name="layout-manager">
+      <object class="GtkBinLayout"/>
+    </property>
+    <child>
+      <object class="GtkDrawingArea" id="drawing_area">
+        <property name="vexpand">1</property>
+        <property name="width-request">600</property>
+        <property name="height-request">600</property>
+      </object>
+    </child>
```

Comment:
I would drop this and override the snapshot vfunc of GtkWidget

Reply:
Thanks, indeed makes it quite a bit simpler